tests/unittest/_torch/auto_deploy/unit/multigpu/transformations/library/test_allreduce_residual_rmsnorm_fusion.py

```
@@ -5,6 +5,7 @@
 from _dist_test_utils import get_device_counts
 from torch.export import export
 
+import tensorrt_llm._torch.auto_deploy  # noqa: F401
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)